scripts/translate_ascii.py

```diff
@@ -55,12 +55,13 @@ def main() -> None:
         get_log().info("ascii file translation complete", duration=time_in_ms() - start_time, count=len(tiffs))
 
         # copy any sidecar files to target
-        sidecars = []
-        start_time = time_in_ms()
-        for ls in asc_files:
-            sidecars += find_sidecars(ls, [".prj", ".tfw"])
-        write_all(inputs=sidecars, target=arguments.target)
-        get_log().info("sidecar files copied", duration=time_in_ms() - start_time, count=len(sidecars))
+        # TODO: TDE-1007 update translate_ascii for new sidecar download method
```

Comment:
This presumably still needs doing?

Reply:
Yes, I put it back in draft as I realised that but you were v. quick :)